frontend/src/components/Board.vue

```diff
@@ -126,7 +135,7 @@ export default class Board extends Vue {
         this.edgeManager.tryConnectVertices(event);
     } else if (selectedTool == "Custom") {
       this.vertexManager.allowDrag();
-      this.handleClick = this.createVertex;
+      this.handleMouseDown = this.createVertex;
```

Comment:
Sadly this won't work, because this `MouseDown` can be called simultaneously with drag event. Try to drag a vertex on the custom mode.